docs/pjrt.md

```diff
@@ -194,15 +194,65 @@ for more information.
 
 *Warning: GPU support is still highly experimental!*
 
+### Single-node GPU training
+
 To use GPUs with PJRT, simply set `PJRT_DEVICE=GPU` and configure
 `GPU_NUM_DEVICES` to the number of devices on the host. For example:
 
 ```
 PJRT_DEVICE=GPU GPU_NUM_DEVICES=4 python3 xla/test/test_train_mp_imagenet.py --fake_data --batch_size=128 --num_epochs=1
 ```
 
-Currently, only a single host is supported, and multi-host GPU cluster support
-will be added in an future release.
+You can also use `torchrun` to initiate the single-node multi-GPU training. For example,
+
+```
+PJRT_DEVICE=GPU torchrun --nnodes 1 --nproc-per-node ${NUM_GPU_DEVICES} xla/test/test_train_mp_imagenet.py --fake_data --batch_size=128 --num_epochs=1
+```
+
+In the above example, `nnodes` means how many machines (phsical machines or VMs) to be used (it is 1 since we do single-node training). `NUM_GPU_DEVICES` means how many GPU devices to be used.
+
+### Multi-node GPU training
```

Comment:
has the multi host GPU pr landed?

Reply:
This is to respond a comment from Will https://github.com/pytorch/xla/pull/5657#issuecomment-1765061981